app/src/main/java/com/example/drawingactivity/DrawActivity.kt

```diff
@@ -32,8 +38,21 @@ class DrawActivity : AppCompatActivity() {
         setContentView(R.layout.activity_main)
         val drawableArea = findViewById<DrawableAreaView>(R.id.drawableArea)
         drawableArea.onBitmapDrawn = { bitmap -> viewModel.setCurrentBitmapState(bitmap) }
-        val colorSelector = findViewById<PrimaryColorCircleView>(R.id.colorCircle)
+        val colorSelector = findViewById<CurrentColorView>(R.id.colorCircle)
         colorSelector.onChangeColor = { viewModel.cycleCurrentColor() }
+        val coordinator = findViewById<CoordinatorLayout>(R.id.coordinator)
+        val bottomSheet = coordinator.findViewById<LinearLayout>(R.id.toolLayout)
+        listOf<View>(
+            findViewById<CoordinatorLayout>(R.id.coordinator),
+            bottomSheet.findViewById<ImageView>(R.id.brushButton),
```

Comment:
`bottomSheet.apply` to remove redundancy

Reply:
using .apply